crates/gourgeist/src/activator/activate.bat

```diff
@@ -0,0 +1,59 @@
+@REM Copyright (c) 2020-202x The virtualenv developers
+@REM
+@REM Permission is hereby granted, free of charge, to any person obtaining
+@REM a copy of this software and associated documentation files (the
+@REM "Software"), to deal in the Software without restriction, including
+@REM without limitation the rights to use, copy, modify, merge, publish,
+@REM distribute, sublicense, and/or sell copies of the Software, and to
+@REM permit persons to whom the Software is furnished to do so, subject to
+@REM the following conditions:
+@REM
+@REM The above copyright notice and this permission notice shall be
+@REM included in all copies or substantial portions of the Software.
+@REM
+@REM THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+@REM EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+@REM MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+@REM NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+@REM LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+@REM OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+@REM WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+@set "VIRTUAL_ENV={{ VIRTUAL_ENV_DIR }}"
+
+@set "VIRTUAL_ENV_PROMPT=venv"
```

Comment:
I think this might mean that the prompt in a cmd shell always has `(venv)` prepended to it when the virtual environment is activated, even when the virtual environment is in a directory with a different name:

![image](https://github.com/astral-sh/uv/assets/66076021/c875bdc2-16ca-44bb-848c-2c69922e62dd)

It _looks_ like the CPython `activate.bat` script has some more complex logic around this (but I know nothing about `.bat` files)

Reply:
Perhaps #1570 addresses this?

Reply:
Right, thanks!